spec/lib/schools_in_area_spec.rb

```diff
@@ -0,0 +1,56 @@
+require 'rails_helper'
+
+RSpec.describe 'SchoolsInArea' do
```

Comment:
We should probably also test what happens when there are more than 25 schools to search.